pkg/ipfs/streamer.go

```diff
@@ -0,0 +1,46 @@
+// VulcanizeDB
```

Comment:
Yeah absolutely!

Reply:
hmm moving both the state diff streamer and the seed node streamer to same libraries/shared/streamer package causes some circular dependency issues since the seed node service needs to import and use the state diff streamer, and the seed node streamer needs to import some packages from the seed node service's package

Reply:
It seems odd to me to move the SeedNodePayload out into the libararies/shared/streamer package (for example, the sated diff streamer does not have its payload struct defined in there- it's defined in the geth state diff package- despite that payload also being used outside of geth) but that is one way to prevent this circular dependency and also addresses your below comment about `ResponsePayload` not being specific to the IPFS package.

Reply:
We should get on a call to discuss this.

Reply:
Just now seeing this, relatively free to hop on calls though.